test/VisionModel/vgg.jl

```diff
@@ -0,0 +1,9 @@
+using Flux
+using FluxModels
+using Test
+
+@testset "VGG" for model in [vgg11, vgg11bn, vgg13, vgg13bn, vgg16, vgg16bn, vgg19, vgg19bn]
+    m = model()
```

Comment:
You need to pass in the `imsize` argument for all VGG constructors.

Reply:
yes, I did while doing the tests manually.

Reply:
Can you commit it to the automated testing though?